certified-connectors/The Bot Platform Connector/apiDefinition.swagger.json

```diff
@@ -0,0 +1,304 @@
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "x-ms-connector-metadata": [
+      {
+        "propertyName": "Website",
+        "propertyValue": "https://thebotplatform.com"
+      },
+      {
+        "propertyName": "Privacy policy",
+        "propertyValue": "https://thebotplatform.com/privacy-policy/"
+      },
+      {
+        "propertyName": "Categories",
+        "propertyValue": "Communication;Website"
+      }
+    ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/v1.0/activity/external": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema":{}
+          }
+        },
+        "summary": "Send a simple text message to a bot user",
+        "operationId": "Simpletextmessage",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "data": {
+                  "type": "object",
+                  "properties": {
+                    "attributes": {
+                      "type": "object",
+                      "properties": {
+                        "messages": {
+                          "type": "array",
+                          "items": {
+                            "type": "object",
+                            "properties": {
+                              "text": {
+                                "type": "string",
+                                "description": "Enter your text message here",
+                                "title": "",
+                                "format": ""
+                              }
+                            }
+                          },
+                          "description": "messages"
+                        },
+                        "recipient": {
+                          "type": "string",
+                          "description": "email address of recipient",
+                          "title": "email address"
+                        }
+                      },
+                      "description": "attributes"
+                    },
+                    "type": {
+                      "type": "string",
+                      "description": "type",
+                      "title": "",
+                      "default": "external-activity",
+                      "enum": [
+                        "external-activity"
+                      ],
+                      "x-ms-visibility": "internal"
+                    }
+                  },
+                  "description": "data",
+                  "required": [
+                    "type"
+                  ]
+                }
+              },
+              "required": [
+                "data"
+              ]
+            }
+          }
+        ],
+        "description": "Send a message to a bot user using text"
+      }
+    },
+    "/": {},
```

Comment:
@tbpsami is it possible to remove the empty operations?

Reply:
Done